src/Tools/DynamoShapeManager/Preloader.cs

```diff
@@ -144,60 +97,6 @@ public Preloader(string rootFolder, IEnumerable<Version> versions)
                 Utilities.GeometryFactoryAssembly);
         }
 
-        /// <summary>
-        /// Constructs a preloader object to help preload the specified version 
-        /// of shape manager from the given directory.
-        /// </summary>
-        /// <param name="rootFolder">Full path of the directory that contains 
-        /// LibG_xxx folder, where 'xxx' represents the library version. In a 
-        /// typical setup this would be the same directory that contains Dynamo 
-        /// core modules. This must represent a valid directory.
-        /// </param>
-        /// <param name="shapeManagerPath">The directory from where shape manager
-        /// binaries can be preloaded from.</param>
-        /// <param name="version">The version of shape manager.</param>
-        /// 
-        [Obsolete("please use Preloader constructor with signature Preloader(string,IEnumerable<Version>)")]
-        public Preloader(string rootFolder, string shapeManagerPath, LibraryVersion version)
-        {
-            if (string.IsNullOrEmpty(rootFolder))
-                throw new ArgumentNullException("rootFolder");
-            if (!Directory.Exists(rootFolder))
-                throw new DirectoryNotFoundException(rootFolder);
-
-            if (string.IsNullOrEmpty(shapeManagerPath))
-                throw new ArgumentNullException("shapeManagerPath");
-            if (!Directory.Exists(shapeManagerPath))
-                throw new DirectoryNotFoundException(shapeManagerPath);
-
-            if (version == LibraryVersion.None)
-                throw new ArgumentOutOfRangeException("version");
-
-            this.Version2 = MapLibGVersionEnumToFullVersion(version);
-            this.ShapeManagerPath = shapeManagerPath;
-
-            var libGFolderName = string.Format("libg_{0}_{1}_{2}", this.Version2.Major, this.Version2.Minor, this.Version2.Build);
-            PreloaderLocation = Path.Combine(rootFolder, libGFolderName);
-            GeometryFactoryPath = Path.Combine(PreloaderLocation,
-                Utilities.GeometryFactoryAssembly);
-        }
-
-        /// <summary>
-        /// Construct a Preloader by specifying a required library version.
-        /// </summary>
-        /// <param name="rootFolder">Full path of the directory that contains 
-        /// LibG_xxx folder, where 'xxx' represents the library version. In a 
-        /// typical setup this would be the same directory that contains Dynamo 
-        /// core modules. This must represent a valid directory.
-        /// </param>
-        /// <param name="version">The version of shape manager.</param>
-        /// <returns></returns>
-        [Obsolete("please use constructor Preloader constructor with signature Preloader(string,IEnumerable<Version>)")]
-#if NET6_0_OR_GREATER
-        [System.Runtime.Versioning.SupportedOSPlatform("windows")]
-#endif
-        public Preloader(string rootFolder, LibraryVersion version)
-            : this(rootFolder, new[] { version }) { }
```

Comment:
Forgive me to ask. Just curious why are we removing so much code here.

Reply:
They are obsolete and we are moving to 3.0.